### PR TITLE
Docs for hardhat and Slither are updated to explain info icon

### PR DESCRIPTION
**Pull request by Aniket-Engg**
fixes https://github.com/ethereum/remix-project/issues/1439

### CODE REVIEW

File: docs/hardhat.md [hunk 1]
```diff
@@ -37,6 +37,8 @@ If a hardhat project is shared through remixd and `localhost` workspace is loade
 
 ![](images/a-hardhat-compilation.png)
 
+There is an info icon along side the label which redirects to a specific section of Remix official documentation that explains how to use Hardhat compilation.
+
 One can check this box to run the compilation for Hardhat along with the Remix using the compiler configuration in `Solidity Compiler` plugin.
```
Comment (ryestew) [inline docs/hardhat.md:42]: With you update - its not so clear what the "this box" is referring to - maybe update it to: 
Check the **Enable Hardhat Compilation** box to ...

File: docs/slither.md [hunk 2]
```diff
@@ -35,6 +35,8 @@ If a project is shared through remixd and `localhost` workspace is loaded in Rem
 
 ![](images/a-slither-analysis.png)
 
+There is an info icon on the right side of the label which redirects to a specific section of Remix official documentation that explains how to use Slither Analysis and prerequisites for it.
+
 One can check this box to run the analysis using Slither along with the Remix's analysis library.
```
Comment (ryestew) [inline docs/slither.md:40]: With you update - its not so clear what the "this box" is referring to - maybe update it to:
Check the Enable Slither Analysis box to ...

**Review by ryestew — CHANGES_REQUESTED** [review]

File: docs/hardhat.md (hunk 1 above)
Comment (Aniket-Engg) [reply docs/hardhat.md:42]: updated

File: docs/slither.md (hunk 2 above)
Comment (Aniket-Engg) [reply docs/slither.md:40]: updated

**Review by ryestew — APPROVED** [review]

**Pull request merged**